client/components/app.login/view.html

```diff
@@ -0,0 +1,15 @@
+<template name="login">
+    {{#if currentUser}}
```

Comment:
Here, this is okay. But it's better if we can use a Flow Component state for this.
We're trying to stop using all the global helpers.